docs/specs/app-initiated-sso-to-web.md

```diff
@@ -0,0 +1,179 @@
+    Note right of Authgear: Alice enter user credentials
+    Authgear->>-app1: [2]Redirect to app://redirect?code=CODE
+    app1->>+Authgear: [3]POST https://auth.example.com/oauth2/token<br/>code=CODE
+    Authgear->>-app1: [4]Response with<br/>access_token=ACCESS_TOKEN<br/>refresh_token=REFRESH_TOKEN<br/>id_token=ID_TOKEN<br/>device_secret=DEVICE_SECRET
+    app1->>+Authgear: [5]POST https://auth.example.com/oauth2/token<br/>grant_type=token-exchange<br/>client_id=2<br/>subject_token_type=urn:ietf:params:oauth:token-type:id_token<br/>subject_token=ID_TOKEN<br/>actor_token_type=urn:x-oath:params:oauth:token-type:device-secret<br/>actor_token=DEVICE_SECRET<br/>requested_token_type=urn:x-oath:params:oauth:token-type:device-browser-session-token
+    Authgear->>-app1: [6]Response with<br/>device_browser_session_token=DEVICE_BROWER_SESSION_TOKEN
+    app1->>Authgear: [7]Open URL In Browser: https://auth.example.com/oauth2/authorize?<br/>client_id=2<br/>id_token_hint=ID_TOKEN<br/>device_browser_session_token=DEVICE_BROWER_SESSION_TOKEN<br/>redirect_uri=https://www.example.com/redirect<br/>state=STATE<br/>prompt=none<br/>response_type=token<br/>response_mode=cookie
+    Authgear->>Web: [8]Redirect to https://www.example.com/redirect?state=STATE<br/>Set Cookie at=ACCESS_TOKEN
+    Note right of Web: Web now authenticated as Alice by cookie
+```
+
+Step [1] to [4] are the standard OpenID Connect authorization_code flow with the following extensions. In step [1], the `x_device_browser_sso` is specified signifying that the client is requesting a `device_secret` to be returned when the code is exchanged for tokens.
+
+After step [4], Mobile App #1 stores the device_secret and id_token in the protected device storage accessible only to Mobile App #1.
+
+Step [5] invokes the /token endpoint with the token exchange profile passing the id_token obtained from the device storage, the client_id and the device secret. A device_browser_session_token is then returned in step [6].
+
+Step [7] uses the obtained device_browser_session_token to open /authorize in a browser, with the id_token used in step [5], and a redirect_uri pointing to Web App #2.
+
+Step [8] redirects the browser to Web App #2, with a session cookies set. The session cookie can then be used for authorizing Web App #2 as it was logged in as the same user of Mobile App #1.
+
+## Native App Authorization Extensions
+
+Same as the [Native SSO](https://openid.net/specs/openid-connect-native-sso-1_0.html#name-native-app-authorization-ex), except that the new scope value supported is `x_device_browser_sso`.
+
+## Token Exchange Profile for App Initiated SSO to Web
+
+This section profiles the OAuth2 Token Exchange [RFC8693] spec and describes the processing rules used to exchange a previous authentication for a device browser session token, which can then be used to exchange for a browser session of another Web App.
```

Comment:
Add back the link to RFC8693 `[RFC8693](https://datatracker.ietf.org/doc/html/rfc8693)`